analyze_inspections.py

```diff
@@ -12,7 +12,10 @@
 import xml.etree.ElementTree as ET
 
 FILE_EXTENSIONS_TO_CONSIDER = [".kt", ".java", ".kts"]
-
+regexes = os.environ.get("INPUT_IGNORE_FILE_PATTERN","NoRegexSpecified")
```

Comment:
Why `NoRegexSpecified`?

Reply:
If the environment variable is not set, it take `None` type. To avoid this, i added a default value.

Reply:
Won't it end up matching against `NoRegexSpecified` then? IMO the correct approach is to take None and only execute the check if value is not None.